help.html

```diff
@@ -87,6 +87,7 @@ <h3>Repositories indexed</h3>
                                              version-control-tools</a></td> <td yes></td>   <td yes></td>   <td yes></td>   <td></td>       <td></td>       <td></td>       </tr>
        <tr><td><a href="/l10n/source/">l10n</a></td>                        <td yes></td>   <td></td>       <td yes></td>   <td></td>       <td></td>       <td></td>       </tr>
        <tr><td><a href="/llvm/source/">LLVM</a></td>                        <td yes></td>   <td yes></td>   <td></td>       <td></td>       <td></td>       <td></td>       </tr>
+       <tr><td><a href="/mingw/source/">MinGW-w64</a></td>                  <td yes></td>   <td yes></td>   <td></td>       <td></td>       <td></td>       <td></td>       </tr>
```

Comment:
mingw-moz should be added here too

Reply:
Ah, oops. Made the change and forgot to squash it into the force-push :) Done now